src/main/java/io/appform/functionmetrics/TraceManager.java

```diff
@@ -0,0 +1,61 @@
+package io.appform.functionmetrics;
+
+import com.google.common.base.Strings;
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import io.opentracing.util.GlobalTracer;
+
+/**
+ * Created by kanika.khetawat on 03/02/21
```

Comment:
Remove author comments and put in some info about hte class

Reply:
Done